README.md

```diff
@@ -1 +1,98 @@
-# private-events
\ No newline at end of file
+# private-events
+
+In this project, we were able to apply all the previously acquired knowledge on Routing, Active Record, Models, Controllers, and some newly acquired knowledge on Advanced Active Record Queries and Associations. The project is very comprehensive and it was a challenge to put everything learned so far together to complete it.
+
+![screenshot1](app/assets/images/screenshot01.png)
+
+![screenshot1](app/assets/images/screenshot02.png)
+
+![screenshot1](app/assets/images/screenshot03.png)
+
+Additional to working with advanced Active Records, we implemented testing using Capibara and RSpec. We included testing for going to the website and making sure the form and fields were there and for the presence of mandatory fields for each class.
+
+![screenshot4](app/assets/images/screenshot04.png)
+
+# Requirements
+
+- Ruby > 2.6.5
+- Rails > 6.0.3
+
+## Getting Started
+
+To get started with the app, cd to the directory where you would like the repo to live by typing on your terminal:
+
+```
+$ cd <directory>
+```
+
+Clone the repo typing:
+
+```
+$ git clone git@github.com:lmaldonadoch/private-events.git
+```
+
+Install the needed gems:
+
+```
+$ bundle install
+```
+
+Finally, migrate the database:
+
+```
+$ rails db:migrate
+```
+
```

Comment:
[OPTIONAL]

You should add the command to run app here as well

Reply:
Run command added :)